pxf/pxf-service/src/scripts/pxf-service

```diff
@@ -72,9 +77,11 @@ function createInstance()
 		return 1
 	fi
 
-	chown -R $instance_owner $instance_root
-	chmod 700 $instance_root/$instance_name
-	return 0
+    if $allow_custom_pxf_user; then
+        chown -R $instance_owner $instance_root
```

Comment:
Should we use chown -R $instance_owner:$instance_owner instead (to include group)?

Reply:
There is no guaranty that that group exists, so we don't have to set it here.